### PR TITLE
Add Strikes to account KP command

### PR DESCRIPTION
**Pull request by ShiningMassXAcc**
Add strikes to game data
Add support for progression checking of single achievement

### CODE REVIEW

**Comment by ShiningMassXAcc** [comment]
Similar question from other PRs - I've not actually done anything with discord bots or python before.  I have not run this, and could use a bit of direction to test or validate (unless, since this is small - you want to run with it).

File: guildwars2/account.py [hunk 1]
```diff
@@ -234,6 +234,15 @@ def is_completed(encounter):
                     if not any(a["id"] == _id and a["done"] for a in results):
                         return "-✖"
                 return "+✔"
+            # Progress toward one achievement has to be reached
+            if encounter["type"] == "progressed_achievement":
+                _id = encounter["id"]
+                _progress = encounter["progress"]
+                for achievement in results:
+                    if achievement["id"] == _id and achievement["current"] >= _progress
```
Comment (Maselkov) [inline guildwars2/account.py:242]: Syntax error

**Review by Maselkov — CHANGES_REQUESTED** [review]
You can find instructions on how to test [here](https://github.com/Maselkov/GW2Bot/pull/89#issuecomment-569926230). I really need to update the readme

Also need to change the line 200 to support `progressed_achievement` type

File: guildwars2/account.py (hunk 1 above)
Comment (ShiningMassXAcc) [reply guildwars2/account.py]: Done

**Comment by ShiningMassXAcc** [comment]
> You can find instructions on how to test [here](https://github.com/Maselkov/GW2Bot/pull/89#issuecomment-569926230). I really need to update the readme
> 
> Also need to change the line 200 to support `progressed_achievement` type

Test instructions helped.  Here's two screenshots from my account (that is noob) and another count that is full clear.

image

image

**Review by Maselkov — APPROVED** [review]
Thank you!

**Pull request merged**